### PR TITLE
feat(STADTPULS-361): New individual and overview page for accounts

### PR DESCRIPTION
**Pull request by vogelino**
This PR adds the overview page of all the sensors as well as the individual account page.

----

This PR does not include:
- The Tokens list and functionality

Those tasks will be done separately

### CODE REVIEW

File: src/components/EditAccountModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  onCancel = () => undefined,
+  onDelete,
+  submitButtonText = "Speichern",
+  cancelButtonText = "Abbrechen",
+  deleteButtonText = "Löschen",
+}) => {
+  const {
+    control,
+    handleSubmit,
+    formState: { errors, dirtyFields },
```
Comment (dnsos) [inline src/components/EditAccountModal/index.tsx:52]: What are `dirtyFields`?

**Review by dnsos — APPROVED** [review]
Fantastic work! Very good additions and improvements!

Thanks a lot for this 🎉

File: src/components/EditAccountModal/index.tsx (hunk 1 above)
Comment (vogelino) [reply src/components/EditAccountModal/index.tsx:52]: Thank you for your review! Sorry for the late reply. Dirty fields are fields whose values have been changed and are unsaved. This terminology comes from the `useForm` hook from the [`react-hook-form`](https://react-hook-form.com/) package that we use to do form validation.

File: src/components/UserInfoHeader/UserInfoWithData.test.tsx [hunk 2]
```diff
@@ -0,0 +1,603 @@
...
+
+describe("Tabs", () => {
+  it("renders no Tokens tab if logged out", () => {
+    useUserData.mockReturnValue(baseUserDataReturn);
+    render(
+      <UserInfoWithData routeAccount={routeAccount} activeTab='sensors' />
+    );
+
+    const tabs = screen.getAllByRole("tab");
+    const tokensTab = screen.queryByRole("tab", {
```
Comment (dnsos) [inline src/components/UserInfoHeader/UserInfoWithData.test.tsx:54]: Really nice that you're testing these with the tab `role`!

**Review by dnsos — APPROVED** [review]
Extremely thorough testing that you added here in the last commits - very well done and really raises the bar for future tests.

**Pull request merged**